.travis.yml

```diff
@@ -19,22 +19,32 @@ jobs:
         - IMAGE_REPO="linux-utils-arm64"
         - TRIVYARCH="ARM64"
         - RELEASE_TAG_DOWNSTREAM=0
+    - os: linux
+      arch: ppc64le
+      env:
+        - IMAGE_REPO="linux-utils-ppc64le"
+        - TRIVYARCH="PPC64LE"
```

Comment:
If we are not using trivy, do we need this environment variable?

Reply:
@akhilerm Yup, that's correct. Have updated the PR. :)